src/tests/unit/containers/orders/order/empty_order/tests/empty-order.spec.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
There is no need in folder "tests" in empty_order cause we are already in src/tests

Reply:
Changed